src/styles/blocks/recommended.scss

```diff
@@ -0,0 +1,27 @@
+.recommended {
+  &__items {
+    display: flex;
+    flex-direction: column;
```

Comment:
looks better decision to use "align-items: center" instead of margins in elements